data/Madhavi210.json

```diff
@@ -1,8 +1,21 @@
 {
   "name": "Madhavi Joshi",
+  "displayStatsPublic": true,
   "type": "personal",
-  "bio": "Learning Full Stack Web Development",
+  "bio": "Newbie in tech | Learning Full Stack Web Development | Learning from Communities & Opensource",
 
+  "tags": ["Open Source", "Web Development", "Frontend","JavaScript"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/Madhavi_Joshi__" },
+    { "icon": "FaGithub", "url": "https://github.com/Madhavi210" },
+    {
+      "icon": "FaLinkedin",
+      "url": "linkedin.com/in/madhavi-joshi-0b3228239/"
+    }
```

Comment:
Hi, Can you make sure the code format is same for the objects?

Reply:
Thank you for your collaboration. There is a GitHub Action that will automatically format the file once merged, so this is not needed, I will mark as resolved